src/servers/src/http.rs

```diff
@@ -990,6 +991,7 @@ impl HttpServer {
                 "/label/{label_name}/values",
                 routing::get(label_values_query),
             )
+            .layer(ServiceBuilder::new().layer(CompressionLayer::new().gzip(true)))
```

Comment:
```suggestion
            .layer(ServiceBuilder::new().layer(CompressionLayer::new()))
```
Looks like the `gzip` option is [enabled by default](https://github.com/tower-rs/tower-http/blob/main/tower-http/src/compression_utils.rs#L108).